packages/stellar-axelar-std/src/string.rs

```diff
@@ -0,0 +1,25 @@
+#[cfg(feature = "alloc")]
+use alloc::vec;
+#[cfg(feature = "alloc")]
+use alloc::vec::Vec;
+
+pub trait StringExt {
+    fn is_ascii(&self) -> bool;
+}
+
+#[cfg(feature = "alloc")]
+impl StringExt for soroban_sdk::String {
+    fn is_ascii(&self) -> bool {
+        const ASCII_RANGE: u8 = 127;
```

Comment:
should be a global constant
```suggestion
        const ASCII_RANGE: u8 = i8::MAX as u8;
```